src/rez/packages.py

```diff
@@ -904,7 +919,23 @@ def get_completions(prefix, paths=None, family_only=False):
     return words
 
 
-def get_latest_package(name, range_=None, paths=None, error=False):
+@overload
+def get_latest_package(name: str, *, range_=None,
+                       paths: list[str] | None = None,
+                       error: Literal[True] = True) -> Package:
```

Comment:
the function returns different types depending on its arguments.  if `error=True` it is always guaranteed to return a `Package`, otherwise it might be `None`.    Not doing this would in many spurious errors about package variables possibly being None.

Reply:
Ah, I missed the fact that the return type was different. Thanks!